theme_beauty/views/snippets/s_product_catalog.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-<template id="s_product_catalog" inherit_id="website.s_product_catalog">
-    <!-- Filter -->
-    <xpath expr="//div[hasclass('container')]" position="before">
-        <div class="o_we_bg_filter bg-white-75"/>
-    </xpath>
-</template>
```

Comment:
Removing the template would require a migration script. Probably it's not worth because, when we'll start working on the themes, we'll likely have to reintroduce the same template anyway 🤷🏻‍♂️
I proposed to keep the template and replace the xpath with something that makes sense for this theme... adjusting the padding a bit maybe ?😉